src/app.ts

```diff
@@ -60,7 +61,8 @@ app.delete(
 
 /** Schema API */
 app.get("/submission/schema/", middleware.wrapAsync(schemaApi.get));
-
+// get template for a given schema
+app.get("/submission/template/:schemaName", middleware.wrapAsync(templateApi.get));
```

Comment:
we also have swagger file to document the endpoints, can you document this api there ? you can use editor.swagger.io to help you

Reply:
the url can be /submission/schema/templates/:schemaName and you can use the schema-api under submission instead of creating template api